app/models/driver.rb

```diff
@@ -0,0 +1,35 @@
+class Driver < ApplicationRecord
+  has_many :trips
+  validates :name, presence: true
+  validates :vin, presence: true
+
+
+  def earned
+    sum = 0
+    trips.each do |trip|
+      if trip.cost < 0
+        return 0
+      else
+        sum += (trip.cost - 165) * 0.8
+      end
+    end
+    return sum.round(2)
```

Comment:
Just a suggestion to use an enumerable. 

```suggestion
    sum = trips.sum do |trip|
     trip.cost < 0 ? 0 : (trip.cost - 165) * 0.8
    end
    
    return sum.round(2)
```